test/e2e/framework/node/wait.go

```diff
@@ -215,7 +215,7 @@ func CheckReadyForTests(c clientset.Interface, nonblockingTaints string, allowed
 		allNodes, err := c.CoreV1().Nodes().List(context.TODO(), opts)
 		if err != nil {
 			e2elog.Logf("Unexpected error listing nodes: %v", err)
-			return false, err
+			return false, nil
```

Comment:
I'm not sure we want to extend retries when we can't even list nodes for the full timeouts allowed for becoming schedulable (sometimes as much as 10 minutes)

maybe return nil to retry if attempts is less than 3 or 5 or something?

Reply:
Made a change that tolerates errors until attempt #4, thx!